draft-irtf-cfrg-vdaf.md

```diff
@@ -799,15 +799,18 @@ Some common functionalities:
 * `concat(parts: list[bytes]) -> bytes` returns the concatenation of the input
   byte strings, i.e., `parts[0] + ... + parts[len(parts)-1]`.
 
-* `from_be_bytes(encoded: bytes) -> int` computes the inverse of
-  `to_be_bytes()`.
+* `from_be_bytes(encoded: bytes) -> int` decodes a big-endian byte string,
```

Comment:
nit: `to_be_bytes` should be defined before `from_be_bytes`

Reply:
This is is alphabetized so that it can be referenced more easily.